commit bbb7444e77451d7e456281655b1102c6da279467
Author: Rob Johnston <rob.johnston@joyent.com>
Date:   2019-04-08T18:49:52+00:00 (6 months ago)
    
    OS-7309 build tools should support loader-based boot images (fix clean target)
    Reviewed by: John Levon <john.levon@joyent.com>
    Approved by: John Levon <john.levon@joyent.com>

diff --git a/tools/format_image/Makefile b/tools/format_image/Makefile
index 9ba19826..e8751be7 100644
--- a/tools/format_image/Makefile
+++ b/tools/format_image/Makefile
@@ -37,4 +37,4 @@ format_image: $(OBJS)
 	$(CC) -c $(CFLAGS) $(CPPFLAGS) -o $@ $<
 
 clean:
-	rm $(TARGETS) $(OBJS)
+	rm -f $(TARGETS) $(OBJS)
